pkg/skaffold/config/options.go

```diff
@@ -35,6 +35,8 @@ type PortForwardOptions struct {
 type SkaffoldOptions struct {
 	ConfigurationFile  string
 	GlobalConfig       string
+	DefaultRepo        string
```

Comment:
is it possible to make this a `*string` so we can differentiate between `nil` and `""`? then we wouldn't need the `DefaultRepoSet` boolean

Reply:
That's what I've tried first but couldn't make it work. I'll try again

Reply:
nope, couldn't find a way to do it